### PR TITLE
chore: update tools configs

### PR DESCRIPTION
**Pull request by yftacherzog**
update tool policies

### CODE REVIEW

**Review by ernesgonzalez33 — APPROVED** [review]

**Comment by scoheb** [comment]
@happybhati is this in line with your migration?

**Comment by ernesgonzalez33** [comment]
@scoheb what migration? Should we hold this PR?

**Comment by scoheb** [comment]
@happybhati is migrating RPs so they target rhtap-releng...he can post the JIRA

**Comment by happybhati** [comment]
Thanks @scoheb ,
@ernesgonzalez33 All the RPA are migrated to `rhtap-releng-tenant` and once this[ PR](https://github.com/redhat-appstudio/tenants-config/pull/252) get merged target will be updated to point to new RPA(Migration [MR](https://gitlab.cee.redhat.com/releng/rhtap-release-data/-/merge_requests/145)).

Jira: [RHTAPREL-723](https://issues.redhat.com/browse/RHTAPREL-723)

**Comment by scoheb** [comment]
@happybhati so if this gets merged, you will adjust the RP here in your PR?

**Comment by happybhati** [comment]
@scoheb I believe we can merge my  PR  https://github.com/redhat-appstudio/tenants-config/pull/252 first(if looks good to you), and any change related to RPA needs to be done here --> https://gitlab.cee.redhat.com/releng/rhtap-release-data/-/tree/main/config/MS/ReleasePlanAdmission/rhtap-o11y?ref_type=heads

CC @yftacherzog @ernesgonzalez33

**Review by scoheb — CHANGES_REQUESTED** [review]
see last comment

**Comment by yftacherzog** [comment]
@scoheb let me know if it's ok now or still missing something.

File: cluster/stone-prd-rh01/managed/rh-managed-rhtap-ser-tenant/release_plan_admission-tools.yaml
[the remark this answers is not in the document]
Comment (yftacherzog) [reply cluster/stone-prd-rh01/managed/rh-managed-rhtap-ser-tenant/release_plan_admission-tools.yaml:1]: Alright!

I removed the RPA from this repo and created an MR to create it where you mentioned it should go.
https://gitlab.cee.redhat.com/releng/rhtap-release-data/-/merge_requests/218

Let me know if that makes sense.

[the remark this answers is not in the document]
Comment (happybhati) [reply cluster/stone-prd-rh01/managed/rh-managed-rhtap-ser-tenant/release_plan_admission-tools.yaml:1]: - I addressed your comment on GitLab [MR ](https://gitlab.cee.redhat.com/releng/rhtap-release-data/-/merge_requests/218)
- The ec-policy o11y use is common for all managed services which is `app-interface-standard`, if require make changes [here ](https://gitlab.cee.redhat.com/releng/rhtap-release-data/-/blob/main/config/MS/EnterpriseContractPolicy/app-interface-standard.yaml?ref_type=heads)

- @scoheb can help with approving/merging this one!

File: cluster/stone-prd-rh01/managed/rh-managed-rhtap-ser-tenant/ec-policy-o11y.yaml
Comment (happybhati) [inline cluster/stone-prd-rh01/managed/rh-managed-rhtap-ser-tenant/ec-policy-o11y.yaml:1]: The [ec-policy](https://gitlab.cee.redhat.com/releng/rhtap-release-data/-/blob/main/config/MS/ReleasePlanAdmission/rhtap-o11y/o11y.yaml?ref_type=heads#L12) your pipeline using is https://gitlab.cee.redhat.com/releng/rhtap-release-data/-/blob/main/config/MS/EnterpriseContractPolicy/app-interface-standard.yaml?ref_type=heads 
common for all managed services.

File: cluster/stone-prd-rh01/managed/rh-managed-rhtap-ser-tenant/release_plan_admission-tools.yaml
[the remark this answers is not in the document]
Comment (yftacherzog) [reply cluster/stone-prd-rh01/managed/rh-managed-rhtap-ser-tenant/release_plan_admission-tools.yaml:1]: The gitlab MR was merged. @scoheb, Can this one be merged?

File: cluster/stone-prd-rh01/managed/rh-managed-rhtap-ser-tenant/ec-policy-o11y.yaml
Comment (ernesgonzalez33) [reply cluster/stone-prd-rh01/managed/rh-managed-rhtap-ser-tenant/ec-policy-o11y.yaml:1]: If that's the case, @yftacherzog you should update this integration test to user the `rhtap-releng-tenant/app-interface-standard` policy and delete the o11y policy here

File: cluster/stone-prd-rh01/tenants/rhtap-o11y-tenant/integration_test_scenario.yaml [hunk 1]
```diff
@@ -53,7 +53,7 @@ spec:
       name: application
   params:
     - name: POLICY_CONFIGURATION
-      value: rh-managed-rhtap-ser-tenant/rh-policy
+      value: rh-managed-rhtap-ser-tenant/rh-policy-o11y
```
Comment (ernesgonzalez33) [inline cluster/stone-prd-rh01/tenants/rhtap-o11y-tenant/integration_test_scenario.yaml:56]: Same here. Point to `rhtap-releng-tenant/app-interface-standard`

**Review by ernesgonzalez33 — CHANGES_REQUESTED** [review]

File: cluster/stone-prd-rh01/managed/rh-managed-rhtap-ser-tenant/ec-policy-o11y.yaml
Comment (yftacherzog) [reply cluster/stone-prd-rh01/managed/rh-managed-rhtap-ser-tenant/ec-policy-o11y.yaml:1]: That policy is used for the o11y repo if I understand correctly. I'd really like to see that I get this merged and working for the tools repo before changing o11y as well.

File: cluster/stone-prd-rh01/tenants/rhtap-o11y-tenant/integration_test_scenario.yaml (hunk 1 above)
Comment (yftacherzog) [reply cluster/stone-prd-rh01/tenants/rhtap-o11y-tenant/integration_test_scenario.yaml]: I changed it. See if this is what you had in mind.

Comment (yftacherzog) [reply cluster/stone-prd-rh01/tenants/rhtap-o11y-tenant/integration_test_scenario.yaml]: @ernesgonzalez33 hi, can you help me get this one merged? It's been open for few weeks and I have a couple of tasks depending on getting that repo properly released.

**Review by ernesgonzalez33 — APPROVED** [review]

**Review by ernesgonzalez33 — APPROVED** [review]

**Review by happybhati — APPROVED** [review]

**Review by scoheb — APPROVED** [review]

**Pull request merged**